src/styles/stars.scss

```diff
@@ -0,0 +1,38 @@
+.catalog {
+  &__card-stars {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    max-width: 166px;
+    height: 16px;
+    margin-top: 16px;
+
+    &-list {
+      display: flex;
+
+      &-star {
+        width: 16px;
+        height: 16px;
+        margin-right: 4px;
+        background: url(../images/star.svg) no-repeat center;
+
+        &:last-child {
+          margin-right: 0;
+        }
+
+        &:nth-child(-n + 4) {
```

Comment:
By using this styling, you are fixing the cards to always be 4 stars active and 1 inactive.
A better approach would be to implement the Stars block the same way you did before on the Stars task.
With that implementation, the modifier for --0, --1... --5 is present at `.stars` class and this changes the actual stars count.
I believe it's a google practice to implement it here and you can do it quite easily by using your own code from Stars as an example ;)

hint: you probably need to check your html too

Reply:
@vinasxz please have a look a this, it is still unresolved!